kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -787,4 +787,14 @@
     });
   }
 
+  var draftbutton = document.getElementsByClassName('btn-draft')[0];
+  var url = $( '.btn-draft' ).data( 'draft-url' );
+  draftbutton.addEventListener('click', function() {
+    var both_data = $('#both_form').serializeArray();
+    var doc_data = $('#doc_form').serializeArray();
+    var rev_data = $('#rev_form').serializeArray();
+    var total_data = both_data.concat(doc_data, rev_data);
+    $.post( url, total_data );
```

Comment:
This should give feedback to users, reporting whether the action was a success or failure.